.github/workflows/publish-nym-vpn-android.yml

```
@@ -1,363 +1,90 @@
+        uses: actions/download-artifact@v8
         with:
-          fileName: ${{ env.KEY_STORE_FILE }}
-          fileDir: ${{ env.KEY_STORE_LOCATION }}
-          encodedString: ${{ secrets.ANDROID_KEYSTORE }}
+          name: ${{ env.UPLOAD_DIR_ANDROID }}
 
-      - name: Create keystore path env var
+      - name: Distribute app to fastlane track ${{ inputs.track }} 🚀
+        env:
+          FASTLANE_OPT_OUT_USAGE: YES
+          ANDROID_NDK_HOME: ${{ steps.setup-ndk.outputs.ndk-path }}
```

actionlint (1.7.12)

[error] property "setup-ndk" is not defined in object type {createserviceaccount: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)